### PR TITLE
added homework (very late :D)

### PR DESCRIPTION
**Pull request by ZamisVlad**

### CODE REVIEW

File: maxmin/max3/max3.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	"fmt"
+	"math"
+)
+
+func main() {
+	fmt.Println("The program finds the maximum of three numbers max(a, b, c).")
+	fmt.Println("Enter three numbers: ")
+	var a, b, c float64
+	fmt.Scan(&a, &b, &c)
+	fmt.Println("Max is", (c+((a+b+math.Abs(a-b))/2)+math.Abs(c-(a+b+math.Abs(a-b))/2))/2)
```
Comment (zayac) [inline maxmin/max3/max3.go:13]: Can you introduce a separate variable for `a+b+math.Abs(a-b))/2` in order to avoid repetitive code?

Same in other programs.

File: resistance/resistance.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("The program calculates the resistance R of two resistors.")
+	var a, b float64
+	fmt.Print("Enter the resistance of two resistors: ")
+	fmt.Scan(&a, &b)
+	fmt.Println("Resistance when connected in series is ", (a + b), ";", "resistance when connected in parallel is", (1 / (1/a + 1/b)))
```
Comment (zayac) [inline resistance/resistance.go:10]: It's better to assign `a + b` and `1 / (1/a + 1/b)` to separate variables.

File: quadratic/quadratic.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+)
+
+func main() {
+	fmt.Println("The program finds solutions of a quadratic equation.")
+	var a, b, c float64
+	fmt.Print("Enter the coefficients A, B and C: ")
+	fmt.Scan(&a, &b, &c)
+	D := b*b - 4*a*c
+	fmt.Println("x1=", (-b+math.Sqrt(D))/(2*a))
+	fmt.Println("x2=", (-b-math.Sqrt(D))/(2*a))
```
Comment (zayac) [inline quadratic/quadratic.go:15]: Now, when you know how to use if conditions, could you implement the program properly, so it handles all corner cases, e.g. `a`, `b`, `c` are zeros, `D < 0`, `D == 0`, etc.

**Comment by zayac** [comment]
Now, when you know how to test the code, could you implement your solution as functions with tests added?
